component/audioinput/v1/audioinput.pb.gw.go

```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/utilities"
-	"go.viam.com/api/common/v1"
+	v1_0 "go.viam.com/api/common/v1"
```

Comment:
These component go file changes are unrelated. Seems like the proto-compiler changed how it imports `go.viam.com/api/common/v1`? 🤷🏻